src/bin/cli.rs

```diff
@@ -112,7 +112,7 @@ fn trimmed(mut vec: Vec<u8>, bytes_read: usize) -> Vec<u8> {
     // Until there is some whatever-newline character, pop.
     while let Some(byte) = vec.last() {
         // Of course, we assume utf-8
-        if byte < &0x0a || byte > &0x0d {
+        if !(&0x0a..=&0x0d).contains(&byte) {
```

Comment:
Wow, really? That's very surprising

Reply:
Ok, that's because a `Range` has a `start_bound()` and `end_bound()` (from `RangeBounds`).

Still even if at least it doesn't need to iterate i find it surprising to instanciate a data structure instead of a simple numeric comparison...